Client.xcodeproj/project.pbxproj

```diff
@@ -5426,12 +5437,23 @@
 				ENABLE_BITCODE = NO;
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
-					"$(PROJECT_DIR)/Carthage/Build/iOS",
 					"$(BUILD_DIR)/Release$(EFFECTIVE_PLATFORM_NAME)",
+					"$(PROJECT_DIR)/Carthage/Build/iOS",
+					"$(PROJECT_DIR)/ThirdParty/PLCrashReporter",
+				);
+				HEADER_SEARCH_PATHS = (
```

Comment:
If we're importing the framework through Carthage we shouldn't need it in ThirdParty too.